easybuild/easyblocks/f/flexiblas.py

```diff
@@ -0,0 +1,107 @@
+##
+# Copyright 2021-2021 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for building and installing FlexiBLAS, implemented as an easyblock
+
+author: Kenneth Hoste (HPC-UGent)
+"""
+import os
+
+from easybuild.easyblocks.generic.cmakemake import CMakeMake
+from easybuild.tools.systemtools import get_shared_lib_ext
+
+
+class EB_FlexiBLAS(CMakeMake):
+    """Support for building/installing FlexiBLAS."""
+
+    def __init__(self, *args, **kwargs):
+        """Easyblock constructor."""
+        super(EB_FlexiBLAS, self).__init__(*args, **kwargs)
+
+        build_dep_names = [dep['name'] for dep in self.cfg.dependencies(build_only=True)]
+        dep_names = [dep['name'] for dep in self.cfg.dependencies()]
+        self.blas_libs = [x for x in dep_names if x not in build_dep_names]
```

Comment:
if you're using sets for both dep_names and build_dep_names, you can do:
```suggestion
        self.blas_libs = dep_names - build_dep_names
```